frontend/src/components/Image/Image.jsx

```diff
@@ -1,23 +1,28 @@
 import React from 'react';
 import { useSearchParams } from "react-router-dom";
 import Comments from "../comment/comment";
+import LikeButton from '../LikesButton/LikesButton';
```

Comment:
Not an issue here but in case you have not come across react-icons, they are great!